src/components/Location/Location.js

```diff
@@ -115,16 +135,46 @@ export const Location = ({ savedTides, setSavedTides, setError, setServerDown })
         >
           {saveMessage}
         </h2>
-      <table className="tide-table">
-        <thead>
-          <tr>
-            <th className="tide-time-l">DATE & TIME</th>
-            <th className="tide-height-l">TIDE</th>
-            <th className="tide-save-head">SAVE?</th>
-          </tr>
-        </thead>
-        <tbody>{renderedTides}</tbody>
-      </table>
+        <table className="tide-table">
+          <thead>
+            <tr>
+              <th className="tide-time-l">DATE & TIME</th>
+              <th className="tide-height-l">TIDE</th>
+              <th className="tide-save-head">SAVE?</th>
+            </tr>
+          </thead>
+          <tbody>{renderedTides}</tbody>
+        </table>
```

Comment:
Maybe a `TidesTable` component would be nice?

Separating the table body into a separate thing does make this smaller but I get a better idea of the table as a whole if it's all in the same location in code.